lib/logstash/plugin_mixins/jdbc/value_handler.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+require "time"
+require "date"
+
+module LogStash module PluginMixins module Jdbc
+  # Provides functions to extract the row's values, ensuring column types
+  # are properly decorated to become coercible to a LogStash::Event.
+  module ValueHandler
+    # Stringify the row keys and decorate values when necessary
+    def extract_values_from(row)
+      Hash[row.map { |k, v| [k.to_s, decorate_value(v)] }]
+    end
+
+    # Decorate the value so it can be used as a LogStash::Event field
+    def decorate_value(value)
+      case value
+      when Time
+        LogStash::Timestamp.new(value)
+      when Date, DateTime
+        LogStash::Timestamp.new(value.to_time)
+      else
+        value
+      end
```

Comment:
Do all tests still pass like this?

```suggestion
      case value
      when Date, DateTime
        value.to_time
      else
        value
      end
```

Reply:
Yes, they still pass with the suggested code. It seems those conversions are not needed anymore, they were probably added [here](https://github.com/logstash-plugins/logstash-integration-jdbc/blob/main/lib/logstash/plugin_mixins/jdbc/jdbc.rb#L263) for previous LS versions  `<= 6.5.0`. Should I change to this new code?

Reply:
Althought performance wise it won't make a difference as the object should be the same, I do suggest we have simpler since the Valuefier already handles Time objects both in 8.x and 7.x